SparseGrids/TasmanianSparseGridWrapC.cpp

```diff
@@ -478,6 +478,56 @@ void tsgGetGPUName(int gpu, int num_buffer, char *buffer, int *num_actual){
 
 void tsgDeleteInts(int *p){ delete[] p; }
 
+void* tsgConstructCustomTabulated(){ return (void*) new CustomTabulated(); }
+void tsgDestructCustomTabulated(void* ct){ delete ((CustomTabulated*) ct); }
+
+void tsgWriteCustomTabulated(void *ct, const char* filename, const bool use_binary_format){
```

Comment:
Write only in ascii format here. The binary format is used when you want to save the whole grid.

Reply:
Done.